pets/etcd/etcd-petset.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apps/v1alpha1
+kind: PetSet
+metadata:
+  name: etcd
+spec:
+  serviceName: "etcd"
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: etcd
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+    spec:
+      containers:
+      - name: etcd
+        image: gcr.io/google_containers/etcd-amd64:2.2.5
+        ports:
+        - containerPort: 2380
+          name: peer
+        - containerPort: 2379
+          name: client
+        env:
+        - name: CLUSTER_SIZE
+          value: "3"
+        - name: PETSET_NAME
+          value: "etcd"
+        volumeMounts:
+        - name: datadir
+          mountPath: /var/run/etcd
+        command:
+          - "/bin/sh"
+          - "-c"
+          - |
+            for i in $(seq 0 $((${CLUSTER_SIZE} - 1))); do
```

Comment:
Won't this stop a re-join from happening for CLUSTER_SIZE > 3 (even if that is hardcoded right now). It feels like this check should only be done on initial cluster setup.